fastapi_template/template/{{cookiecutter.project_name}}/conditional_files.json

```diff
@@ -56,13 +56,18 @@
             "alembic.ini",
             "{{cookiecutter.project_name}}/web/api/dummy",
             "{{cookiecutter.project_name}}/web/gql/dummy",
-            "{{cookiecutter.project_name}}/db_sa",
```

Comment:
Please place this resource back. It will leave db_sa folder in a proejct, if database is disabled.

Reply:
Or maybe not, because of SQLAlchemy. Selector. I will check it.

Reply:
The last review suggested removing it. I tried running this with SQLAlchemy and it seemed fine.